test

```diff
@@ -41,10 +41,10 @@ if [[ ! -d $dir/.build/ ]]; then
 	exit 1
 fi
 
-cname="$(basename "$(realpath "$dir/.build/$1")" .artifacts)"
+cname="$(./build --resolve-cname "$1")"
```

Comment:
is there a reason to remove the basename/realpath thing? I think that is needed for calling the script from another directory.